crates/ln-dlc-node/src/tests/add_dlc.rs

```diff
@@ -1,4 +1,98 @@
-#[test]
-fn given_sibling_channel_when_dlc_channel_present_then_can_create_dlc() {
-    // todo
+use crate::node::Node;
+use crate::tests::dummy_contract_input;
+use crate::tests::init_tracing;
+use crate::tests::wait_until;
+use anyhow::anyhow;
+use bitcoin::Amount;
+use dlc_manager::sub_channel_manager::SubChannelState;
+use dlc_manager::Oracle;
+use dlc_manager::Storage;
+use std::time::Duration;
+
+#[tokio::test]
+#[ignore]
```

Comment:
Is this being ignored on purpose?

Reply:
I think the right approach for these tests is to:

- Ignore them by default (at least until we mock the dependencies).
- Only run them (`cargo test -p ln-dlc-node -- --ignored`) on CI if `ln-dlc-node` (and/or other dependencies) actually change.

Were we actually running them on CI before this PR?

Reply:
> Were we actually running them on CI before this PR?

I've confirmed that we were _not_ running them on CI before this PR.

Nonetheless, I've created an issue that I will tackle right after merging this PR: #157.

Reply:
Ended up doing it with this PR.

Reply:
I initially solved it with a feature flag. What's the reason why you didn't go with this approach?

Reply:
I didn't like the feature flag because the tests aren't even _built_ with `cargo clippy --all-targets`.

I think using the `#[ignore]` attribute to ignore tests is much more predictable too.